### PR TITLE
Markdown Styling

### PR DESCRIPTION
**Pull request by hxltrhuxze**
## Overview

This pull request aims to provide styling for generated markdown. It also contains fixes for components that suffer style-bleeding from the markdown styles.

## Work carried out

- [x] Styled the default markdown content
- [x] Added a header to the post article that is generated via the `title` and `description` front matter
- [x] Updated the code block to remove the grey border and make it sit better in the page
- [x] Tweaked the colour palette of the code block so that it matches the Standards colour palette more closely.
- [x] Fixed the Sketch widget that had style-bleed from the markdown styles.
- [x] Fixed the Hero Banner that had style-bleed from the markdown styles.
- [x] Fixed the Footer that had style-bleed from the markdown styles.
- [x] Additional styling cleanup of the Hero Banner & Footer.

### CODE REVIEW

File: packages/css-framework/src/components/_container.scss [hunk 1]
```diff
@@ -2,16 +2,16 @@
   max-width: $navy-content-width;
   margin-left: auto;
   margin-right: auto;
-  padding-left: spacing(3);
-  padding-right: spacing(3);
```
Comment (antictor) [inline packages/css-framework/src/components/_container.scss:6]: Wow thats a big change

Comment (hxltrhuxze) [reply packages/css-framework/src/components/_container.scss:6]: `spacing(3)` is `0.75rem`. That's nowhere near enough padding.

Comment (hxltrhuxze) [reply packages/css-framework/src/components/_container.scss:6]: If we think we need it that small for apps, then I can put it back. _However_, we categorically need more padding on the docs site, so I'd suggest we create a completely separate container for the docs.

**Review by m7kvqbe1 — CHANGES_REQUESTED** [review]
Slight style bug here on focus state:
Screenshot 2019-07-24 at 14 43 47

File: packages/docs-site/src/components/presenters/Links/Links.scss [hunk 2]
```diff
@@ -3,16 +3,26 @@
...
   display: inline-block;
-  padding: 0 spacing(2);
-  border-left: 2px solid;
+  padding: 0 spacing(4);  
   font-size: font-size(s);
+  margin-bottom: 0;
+  position: relative;
+  &::after {
+    content: "•";
+    color: color(primary, 700);
```
Comment (m7kvqbe1) [inline packages/docs-site/src/components/presenters/Links/Links.scss]: The colour choice here and with the hr seems a little odd to me? I prefer a white / grey?

**Review by m7kvqbe1 — CHANGES_REQUESTED** [review]
Phase banner now needs explicit styling for anchor (it lost the `text-decoration: underline`):

Screenshot 2019-07-24 at 14 51 15

**Comment by m7kvqbe1** [comment]
Slight issue with DataTable headings (only really notice on focus state):

Screenshot 2019-07-24 at 14 56 38

**Comment by m7kvqbe1** [comment]
> Phase banner now needs explicit styling for anchor (it lost the `text-decoration: underline`):
> 
> Screenshot 2019-07-24 at 14 51 15

This has now been resolved.

**Comment by m7kvqbe1** [comment]
H3 from markdown that sits directly above a DataTable could do with some spacing:

Screenshot 2019-07-24 at 15 15 22

**Review by m7kvqbe1 — APPROVED** [review]

**Pull request merged**